src/components/Offline/Offline.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+const Offline = () => (
+  <div>
+    <header style={{ height: '84px' }}></header>
+    <main style={{ display: 'block', widht: '100%' }}>
+      <section class="section__block image__main">
+        <div class="wrapper" style={{ width: '800px', margin: 'auto' }}>
+          <h1 style={{ 'padding-bottom': '0' }}>Experimentamos dificultades técnicas.</h1>
+          <h2
+            style={{
```

Comment:
we dont have css classes for that?

Reply:
I'm not sure this is merely a draft, a transcription of the html that we've been given.